tests/test_verilog/test_bind.py

```diff
@@ -64,12 +65,13 @@ class Main(m.Circuit):
         assert not os.system('cd tests/test_verilog/build && '
                              'verilator --lint-only bind_uniq_test.v '
                              'RTLMonitor.sv RTLMonitor_unq1.sv -Wno-MODDUP'
-                             ' --top-module foo_Main')
+                             ' --top-module bar_foo_Main')
         # TODO: For now, we ignore duplicate modules since each monitor will
         # regenerate coreir primitives.  # since these have the same definition
         # it's okay, but in general this is a bad flag to have on because if
         # you do have duplicate modules with different definitions, you'd want
         # to catch that
-        # When we merge the inline verilog module branch, this will be resolved since
-        # bind modules can be represented explicitly in the graph and compiled
-        # by coreir, so only one definition per primitive will be emitted
+        # When we merge the inline verilog module branch, this will be resolved
```

Comment:
is this an outdated comment? "When we merge the inline..." seems like it would be outdated?

Reply:
I've updated the comment, this isn't resolved by inline verilog because bind can't use it (the bind statement appears outside of a module, whereas inline verilog only supports verilog inside a module definition).  I think when the comment was written I was thinking that bind would use inline verilog but for now we can't do it (we need to add an explicit bind construct to coreir)